src/main/Catalog.js

```diff
@@ -3,7 +3,7 @@ import InfiniteScroll from "react-infinite-scroll-component";
 import SearchForm from "./SearchForm";
 import "./Catalog.css";
 
-const beersUrl = `https://api.punkapi.com/v2/beers`;
+export const beersUrl = `https://api.punkapi.com/v2/beers`;
```

Comment:
You should consider creating a top-level `constants.js` file and put such declarations there.